openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -49,7 +49,7 @@ def _add_waffle_switch_instances(self, switches_dict):
         """
         waffle_switch_instances = WaffleSwitch.get_instances()
         for switch_instance in waffle_switch_instances:
-            switch_name = switch_instance.namespaced_switch_name
+            switch_name = switch_instance.name
```

Comment:
Nit: You probably don't need a variable, since `switch_instance.name` is short enough and read nicely.